.github/workflows/build-m1-binaries.yml

```diff
@@ -60,12 +79,11 @@ jobs:
           name: torchvision-py${{ matrix.py_vers }}-macos11-m1
           path: dist/
       - name: Upload wheel to S3
-        if: ${{ github.event_name == 'push' && steps.extract_branch.outputs.branch == 'nightly' }}
+        if: ${{ github.event_name == 'push' && (steps.extract_branch.outputs.branch == 'nightly' || startsWith(github.event.ref, 'refs/tags/')) }}
         shell: arch -arch arm64 bash {0}
         env:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_PYTORCH_UPLOADER_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_PYTORCH_UPLOADER_SECRET_ACCESS_KEY }}
-          CHANNEL: nightly
```

Comment:
Err, this will fail for nightly builds, isn't it (I.e. `Set CHANNEL (only for tagged pushes)` would only set it to test, but what will set it to nightly?

Reply:
please see comment below we do set CHANNEL nightly by default